submodules/groups/groups.js

```diff
@@ -1235,6 +1235,9 @@ define(function(require) {
 								}
 							});
 
+							//Hide delay column if ring strategy is set to 'In order'
+							strategy === 'single' ? $('.options .option.delay', popup_html).hide() : '';

```

Comment:
Why not use an `if` statement here?

Ternaries should be used to resolve a condition into a value, not into a statement.

Reply:
Ahh good point